packages/react/src/components/primitive/allowed-content.tsx

```
@@ -0,0 +1,46 @@
+  { children, as, ...props }: PrimitiveAllowedContentProps<TTag>,
+  ref: Ref<HTMLDivElement>,
+) {
+  const { routeConfig, state } = usePrimitiveValues("AllowedContent");
+
+  const Comp = as ?? DEFAULT_ALLOWED_CONTENT_TAG;
+
+  return (
+    <Comp data-state={state} {...props} ref={ref}>
+      <PrimitiveSlot
+        default={allowedContentTextLabelGenerator(routeConfig)}
+        children={children}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)